code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,34 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons))
+			var/list/contents = W.contents
+			var/crayoncount = 0
+			for(var/C in contents)
+				if(istype(C, /obj/item/toy/crayon))
+					crayoncount += 1
+				continue
```

Comment:
This `continue` statement isn't necessary. The loop automatically continues at the end of its block unless you `break` it.

You don't really need the loop, though. You're dealing with a container that's only allowed to contain crayons, and want to know if it's full. You can simply compare the quantity of contents to its maximum capacity:
`if(W.contents.len == W.storage_slots)`

As an added bonus, you'll no longer need to assume a box of crayons holds exactly 6 of them.